common/build/build-common/api-extractor-report.esm.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "./api-extractor-base.esm.json",
+	// Use dummy value, forcing all uses to specify the appropriate (broadest of public.d.ts, beta.d.ts,
```

Comment:
This comment belongs with the `.current` case. It would be fine if the main entrypoint value here were something else or just left out here.